external/daint_venv/install.sh

```diff
@@ -40,23 +36,6 @@ if [ $save_wheel ]; then
 fi
 python3 -m pip install --find-links=$wheeldir cupy Cython clang-format
 
-
-# installation of gt4py
-rm -rf gt4py
-git clone git@github.com:ai2cm/gt4py.git gt4py
-cd gt4py
-if [ -z "${GT4PY_VERSION}" ]; then
-    wget 'https://raw.githubusercontent.com/ai2cm/pace/main/GT4PY_VERSION.txt'
-    GT4PY_VERSION=`cat GT4PY_VERSION.txt`
-fi
-git checkout ${GT4PY_VERSION}
```

Comment:
removing this checkout of the version means it is not really a relevant variable anymore, correct?
The only use case I see now is for saving /  fetching caches. Does it still make sense to have this as a global variable?

Reply:
Yes, I've deleted this everywhere except the fetch/generate caches scripts. This was actually very helpful because it showed me another place we had duplicated the fetch caches logic (which I've now removed).